### PR TITLE
Add Vue Integration Guide

### PR DESCRIPTION
**Pull request by chris-visser**
This is the start of the VueJS integration guide for Meteor. I'm planning to type the rest of the guide topics and also adopted the structure a bit from the React guide.

### CODE REVIEW

File: content/vue.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+
+<h3 id="integrating-vue-with-meteor">Integrating Vue With Meteor</h3>
+
+To install Vue in Meteor 1.8, you should add it as an npm dependency:
+
+```sh
+meteor npm install --save vue
+```
+
+At time of writing this guide, Vue is not yet a first class citizen of Meteor and therefore it requires you to install one extra Meteor package to support Vue.
```
[the remark this answers is not in the document]
Comment (chris-visser) [reply content/vue.md]: I will remove it. Thanks for the feedback

**Comment by lorensr** [comment]
Thanks Chris! Is this a WIP PR or would you like it to be merged as-is, with the TODOs left in and worked on later?

**Comment by chris-visser** [comment]
Hi, I would like it to be merged since it should already help people get started.

**Comment by lorensr** [comment]
Looks good!

image

**Pull request merged**